information-schema/information-schema-tidb-trx.md

```diff
@@ -65,73 +70,83 @@ select * from information_schema.tidb_trx\G
-| INSTANCE           | varchar(64)                                             | YES  |      | NULL    |       |
-| ID                 | bigint(21) unsigned                                     | NO   | PRI  | NULL    |       |
-| START_TIME         | timestamp(6)                                            | YES  |      | NULL    |       |
-| CURRENT_SQL_DIGEST | varchar(64)                                             | YES  |      | NULL    |       |
-| STATE              | enum('Normal','LockWaiting','Committing','RollingBack') | YES  |      | NULL    |       |
-| WAITING_START_TIME | timestamp(6)                                            | YES  |      | NULL    |       |
-| MEM_BUFFER_KEYS    | bigint(64)                                              | YES  |      | NULL    |       |
-| MEM_BUFFER_BYTES   | bigint(64)                                              | YES  |      | NULL    |       |
-| SESSION_ID         | bigint(21) unsigned                                     | YES  |      | NULL    |       |
-| USER               | varchar(16)                                             | YES  |      | NULL    |       |
-| DB                 | varchar(64)                                             | YES  |      | NULL    |       |
-| ALL_SQL_DIGESTS    | text                                                    | YES  |      | NULL    |       |
-+--------------------+---------------------------------------------------------+------+------+---------+-------+
+*************************** 1. row ***************************
+             id: 426789913200689153
+all_sql_digests: ["e6f07d43b5c21db0fbb9a31feac2dc599787763393dd5acbfad80e247eb02ad5","04fa858fa491c62d194faec2ab427261cc7998b3f1ccf8f6844febca504cb5e9","b83710fa8ab7df8504920e8569e48654f621cf828afbe7527fd003b79f48da9e"]
+       all_sqls: ["begin","insert into `t` values ( ... )","update `t` set `v` = `v` + ?"]
+*************************** 2. row ***************************
+             id: 426789921471332353
+all_sql_digests: ["e6f07d43b5c21db0fbb9a31feac2dc599787763393dd5acbfad80e247eb02ad5","38b03afa5debbdf0326a014dbe5012a62c51957f1982b3093e748460f8b00821"]
+       all_sqls: ["begin","update `t` set `v` = `v` + ? where `id` = ?"]
 ```
 
-## SQL Digest
+This query calls the [`TIDB_DECODE_SQL_DIGESTS`](/functions-and-operators/tidb-functions.md#tidb_decode_sql_digests) function on the `ALL_SQL_DIGESTS` column of the `TIDB_TRX` table, and converts the SQL digest array into an array of normalized SQL statement through the system's internal query. This helps you visually obtain the information of the statements that have been historically executed by the transaction. However, note that the above query scans the entire table of `TIDB_TRX` and calls the `TIDB_DECODE_SQL_DIGESTS` function for each row. Calling the `TIDB_DECODE_SQL_DIGESTS` function has a high overhead. Therefore, if many concurrent transactions exist in the cluster, try to avoid this type of query.
```

Comment:
```suggestion
This query calls the [`TIDB_DECODE_SQL_DIGESTS`](/functions-and-operators/tidb-functions.md#tidb_decode_sql_digests) function on the `ALL_SQL_DIGESTS` column of the `TIDB_TRX` table, and converts the SQL digest array into an array of normalized SQL statement through the system internal query. This helps you visually obtain the information of the statements that have been historically executed by the transaction. However, note that the above query scans the entire table of `TIDB_TRX` and calls the `TIDB_DECODE_SQL_DIGESTS` function for each row. Calling the `TIDB_DECODE_SQL_DIGESTS` function has a high overhead. Therefore, if many concurrent transactions exist in the cluster, try to avoid this type of query.
```

Reply:
对于客观事物，能避免 's 可以尽量避免